imdclient/IMDClient.py

```diff
@@ -112,13 +112,15 @@ def get_imdframe(self):
             except EOFError:
                 # in this case, consumer is already finished
                 # and doesn't need to be notified
+                self._wait(0)
```

Comment:
Why is "0" hardcoded instead of allowing user to request waiting behavior with kwarg?